Lib/threading.py

```diff
@@ -1197,6 +1207,17 @@ def is_alive(self):
         assert not self._is_stopped and self._started.is_set()
         return True
 
+    def isAlive(self):
```

Comment:
No, please don't add new methods to stable branches (3.7). Revert this change from your PR.